diff --git a/node_modules/@vuepress/plugin-docsearch/lib/client/composables/useDocsearchShim.js b/node_modules/@vuepress/plugin-docsearch/lib/client/composables/useDocsearchShim.js
index 7bb49f3..d16147d 100644
--- a/node_modules/@vuepress/plugin-docsearch/lib/client/composables/useDocsearchShim.js
+++ b/node_modules/@vuepress/plugin-docsearch/lib/client/composables/useDocsearchShim.js
@@ -43,7 +43,9 @@ export const useDocsearchShim = () => {
         navigator: {
             // when pressing Enter without metaKey
             navigate: ({ itemUrl }) => {
-                router.push(itemUrl);
+                // TODO: 暂时解决 algolia 搜索时, 回车跳转失败的问题
+                window.location.href = itemUrl
+                // router.push(itemUrl);
             },
         },
         // add search debounce
